timed/employment/filters.py

```diff
@@ -53,13 +54,20 @@ class UserFilterSet(FilterSet):
     active      = NumberFilter(field_name='is_active')
     supervisor  = NumberFilter(field_name='supervisors')
     is_reviewer = NumberFilter(method='filter_reviewers')
+    is_supervisor = NumberFilter(method='filter_supervisors')
 
     def filter_reviewers(self, queryset, name, value):
         reviewer = Project.objects.filter(reviewers=OuterRef('pk'))
         return queryset.annotate(
             is_reviewer=Exists(reviewer)
         ).filter(is_reviewer=value)
 
+    def filter_supervisors(self, queryset, name, value):
+        supervisor = User.objects.filter(supervisors=OuterRef('pk'))
```

Comment:
This does exactly do what you expect I think. It would return all users which have a supervisor.

I have just found that there are actually methods in UserManager for `all_reviewers` and `all_supervisors`.

This could then look something like this `queryset.filter(pk__in=User.objects.all_supervisors)`.